### DISCUSSION TITLE
Can't get web content

### DISCUSSION THREAD

**Discussion by kreativnix** [curl/curl#14088] [Q&A] [answered]
It should be as easy:
$ curl https://dmraustria.at/ipsc2-oe-master-relais-dashboard/monitor -o dmr.txt
If you try then link in your browser you see a table.
But DMR.txt is empty.
Also using  "--user-agent" with appropriate value did not work.

How can I get this web content?

**Comment by bagder** [comment, marked as answer]
## Check if it is a redirect

curl -I $URL

returns a 301/302 response code, then you need to add `--location` to follow the redirect

## Maybe it needs cookies

As the previous command shows, it sets cookies before the redirect so maybe it wants them passed back in the next request, also enable those:

    curl --location --cookie "" https://dmraustria.at/ipsc2-oe-master-relais-dashboard/monitor -o dmr.txt
